src/main/java/book/store/repository/CategoryRepository.java

```diff
@@ -5,4 +5,5 @@
 import org.springframework.stereotype.Repository;
 
 @Repository
```

Comment:
If you use this annotation (that is not necessary), use it for all repositories, be consistent